src/Page.php

```diff
@@ -69,6 +70,60 @@ public function __construct(Target $target, array $frameTree)
         $this->frameManager = new FrameManager($this, $frameTree);
     }
 
+    /**
+     * Emulates given device metrics and user agent. This method is a shortcut  
+     * for calling two methods: `Page::setUserAgent()` and `Page::setViewport()`    
+     * To aid emulation, Chrome provides a list of device descriptors that can  
+     * be obtained via the `BrowserFactory::getDevice()`. `Page::emulate()`  
+     * will resize the page. A lot of websites don't expect phones to change size,  
+     * so you should emulate before navigating to the page.  
+     * 
+     * Usage:
+     * 
+     * ```php
+     * use HeadlessChromium\BrowserFactory;
+     * 
+     * $browserFactory = new BrowserFactory();
+     * 
+     * // starts headless chrome
+     * $browser = $browserFactory->createBrowser();
+     * 
+     * // get the device for emulation
+     * $iphone = $browserFactory->getDevice('Iphone 6');
+     * 
+     * // specify device positioning 
+     * $iphone->landscape();
```

Comment:
Perhaps it could be useful to return $this from Device::class instance to achieve that syntax:

- $page->emulate($iphone->landscape());
- $page->emulate($iphone->portrait());